css/index.css

```diff
@@ -123,6 +123,15 @@
   display: grid;
   grid-template-rows: repeat(2, 1fr);
   grid-template-columns: 1fr;
+  background-color: white;
```

Comment:
maybe, we should define a variable for the colour `white`?

Reply:
resolved in https://github.com/innoq/SCS/pull/64/commits/c27c5e26a1d6db629d3c6579b916caeda7985261